OpenRA.Mods.Common/Widgets/SupportPowerTimerWidget.cs

```diff
@@ -19,47 +19,165 @@ namespace OpenRA.Mods.Common.Widgets
 {
 	public class SupportPowerTimerWidget : Widget
 	{
+		public readonly int XObserver = 10;
```

Comment:
Or there could be a container widget in ingame.yaml that has all the fields except X and Y. And SupportPowerTimer could be in both player and observer yamls with respective X and Y values...